addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -41,6 +41,8 @@
                         <!-- Date & Time -->
                         <div class="o_wevent_sidebar_block">
                             <h6 class="o_wevent_sidebar_title">Date &amp; Time</h6>
+                            <div itemprop="startDate" class="d-none" t-field="event.date_begin"/>
+                            <div itemprop="endDate" class="d-none" t-field="event.date_end"/>
```

Comment:
- Would use <meta> instead of a d-none div, as it seems more "standard"
- Would include the timezone of the event
- Would use t-out with explicit `isoformat()` call in t-out instead of t-field as it's purely technical (stop me if I'm wrong), so we don't need/want fancy formatting options to get in the way

Reply:
thanks for the feedback :slightly_smiling_face:  Just a minor thing but I think using `isformat()` is not necessary since the date is already in the correct format.

Reply:
I feel like with isoformat you wouldn't need to add "Z" manually but this seems ok